src/javaMain/kotlin/io/github/oshai/kotlinlogging/slf4j/Slf4jExtensions.kt

```diff
@@ -25,3 +30,150 @@ public fun KotlinLogging.logger(underlyingLogger: Logger): KLogger =
   Slf4jLoggerFactory.wrapJLogger(underlyingLogger)
 
 public fun Logger.toKLogger(): KLogger = KotlinLogging.logger(this)
+
+/** Lazy add a log message if isTraceEnabled is true */
+public fun KLogger.trace(
+  throwable: Throwable? = null,
+  marker: org.slf4j.Marker?,
```

Comment:
Maybe I missed something, but why those methods needs to be added?
Anyway Marker.toSlf4j() will treat it correctly when the actual call happens, isn't it?

Reply:
I've added those extension methods as a convenience, so the user doesn't have to convert their `org.slf4j.Marker` to `io.github.oshai.kotlinlogging.Marker` manually.

If you think this is not desired, I can remove them from this PR.

What essentially happens is `marker.toKotlinLogging().toSlf4j()` which is equivalent to `marker` for the call to the underlying Logger.
(And if the underlying Logger is not Slf4j, it still behaves as a normal marker using the implementation of `getName()` from `org.slf4j.Marker`.)

Reply:
got it. I think I prefer to keep the api slim, not adding those extension methods. So it's better to remove from the PR.

Reply:
I've updated the PR accordingly.